src/test/java/org/reaktivity/nukleus/http/internal/util/HttpUtilTest.java

```diff
@@ -81,23 +81,126 @@ public void shouldHandleAllHyphensHeaderName()
     }
 
     @Test
-    public void shouldAcceptValidPath()
+    public void shouldAcceptPathWithAllValidCharacters()
+    {
+        String path = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-._~:/?#[]@!$&'()*+,;=%";
+        assertTrue(HttpUtil.isPathValid(new UnsafeBuffer(path.getBytes())));
```

Comment:
Change this to:
```
byte[] ascii = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-._~:/?#[]@!$&'()*+,;=%".getBytes(US_ASCII);
assertTrue(HttpUtil.isPathValid(new UnsafeBuffer(ascii)));
```
where `US_ASCII` is a static import of `StandardCharsets.US_ASCII`.

Please apply this change to all the test methods.